### PR TITLE
Update TestApp to use INT apps instead of prod

### PR DESCRIPTION
**Pull request by richa-d**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

The TestApp for React Native was currently using apps from the prod portal. We changed TestApp to use apps from the INT portal.

## Related PRs or issues

[#69902](https://msmobilecenter.visualstudio.com/Mobile-Center/_boards/board/t/Amit-Team/Backlog%20Items/?workitem=69902)

### CODE REVIEW

File: TestApp/android/app/src/main/java/com/testapp/MainApplication.java [hunk 1]
```diff
@@ -45,6 +47,9 @@ public ReactNativeHost getReactNativeHost() {
     @Override
     public void onCreate() {
         super.onCreate();
+        AppCenter.setLogUrl("https://in-integration.dev.avalanch.es");
+        Data.setTokenExchangeUrl("https://token-exchange-mbaas-integration.dev.avalanch.es/v0.1");
+        Auth.setConfigUrl("https://config-integration.dev.avalanch.es");
         AppCenter.setLogLevel(Log.VERBOSE);
```
Comment (guperrot) [inline TestApp/android/app/src/main/java/com/testapp/MainApplication.java]: Move log level first

File: TestApp/ios/TestApp/AppDelegate.m [hunk 2]
```diff
@@ -23,10 +23,15 @@
...
+@import AppCenterData;
+@import AppCenterAuth;
 
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [MSAppCenter setLogUrl:@"https://in-integration.dev.avalanch.es"];
+  [MSData setTokenExchangeUrl:@"https://token-exchange-mbaas-integration.dev.avalanch.es/v0.1"];
+  [MSAuth setConfigUrl:@"https://config-integration.dev.avalanch.es"];
   [MSAppCenter setLogLevel:MSLogLevelVerbose];
```
Comment (guperrot) [inline TestApp/ios/TestApp/AppDelegate.m]: move log level first

File: TestApp/android/app/src/main/java/com/testapp/MainApplication.java (hunk 1 above)
Comment (richa-d) [reply TestApp/android/app/src/main/java/com/testapp/MainApplication.java]: Done.

File: TestApp/ios/TestApp/AppDelegate.m (hunk 2 above)
Comment (richa-d) [reply TestApp/ios/TestApp/AppDelegate.m]: Done.

File: TestApp/android/app/src/main/java/com/testapp/MainApplication.java [hunk 3]
```diff
@@ -12,6 +12,8 @@
 import com.facebook.react.ReactPackage;
 import com.facebook.soloader.SoLoader;
 import com.microsoft.appcenter.AppCenter;
+import com.microsoft.appcenter.data.Data;
```
Comment (dhei) [inline TestApp/android/app/src/main/java/com/testapp/MainApplication.java]: sort imports

File: TestApp/ios/TestApp/AppDelegate.m [hunk 4]
```diff
@@ -23,11 +23,16 @@
 #import <AppCenterReactNativeShared/AppCenterReactNativeShared.h>
 
 @import AppCenter;
+@import AppCenterData;
+@import AppCenterAuth;
```
Comment (dhei) [inline TestApp/ios/TestApp/AppDelegate.m]: sort imports.

Comment (richa-d) [reply TestApp/ios/TestApp/AppDelegate.m]: done

File: TestApp/android/app/src/main/java/com/testapp/MainApplication.java (hunk 3 above)
Comment (richa-d) [reply TestApp/android/app/src/main/java/com/testapp/MainApplication.java]: done

**Pull request merged**